mono_repo/test/root_config_test.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
+
+import 'package:mono_repo/src/commands/github/generate.dart';
+import 'package:mono_repo/src/root_config.dart';
+import 'package:path/path.dart' as path;
+import 'package:test/test.dart';
+
+void main() {
+  group('RootConfig', () {
+    test('parseActionVersions', () {
+      final file = File(path.join('..', defaultGitHubWorkflowFilePath));
+      final parsedVersions = RootConfig.parseActionVersions(file);
```

Comment:
Looking at this api, its a bit weird for it to take a `File` - I would probably make it take a `String` instead, which makes it a bit more general (I also want to use this API from a builder which can read the versions out, but that can't give a File).

Reply:
Actually I realize now I can't use a normal builder since this file lives outside of any package anyways 😭

Reply:
👍 - changed